ucsc/ucsc-news-functionality
# Pre-launch clean-up, fixes, and build tooling

See commits for details of all changes

## Testing
1. Clone this PR
2. Run `npm i` and then `npm run zip`
3. Unpack the zipped plugin in a different WordPress site
4. Activate it and test expected features
